lib/MusicBrainz/Server/EditSearch/Query.pm

```diff
@@ -67,7 +67,7 @@ has combinator => (
 );
 
 has order => (
-    isa => enum([qw( asc desc rand )]),
+    isa => enum([qw( asc desc closedasc closeddesc expiringasc expiringdesc rand )]),
```

Comment:
1. The term “expiring” should probably be replaced with “vote closing” since MBS-9732.
2. Aggregated values could be replaced with separate `sort_by` query parameter, or at the very least be more readable using a `-` separator.

Reply:
Fair enough. sort_by seems annoying but - does make it much more clear :)